src/Aspire.Dashboard/Resources/Dialogs.resx

```diff
@@ -263,9 +263,18 @@
     <comment>{0} is the field name, whereas {1} is the length. We don't want to show the field name, so it is missing.</comment>
   </data>
   <data name="SettingsDialogDashboardLogsAndTelemetry" xml:space="preserve">
-    <value>Dashboard logs and telemetry</value>
+    <value>Resource logs and telemetry</value>
   </data>
   <data name="SettingsRemoveAllButtonText" xml:space="preserve">
     <value>Remove all</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="SettingsDialogEnableTelemetry" xml:space="preserve">
+    <value>Enable dashboard telemetry</value>
+  </data>
+  <data name="SettingsDialogTelemetryEnabledInfo" xml:space="preserve">
+    <value>When enabled, Aspire will collect usage data and send it to Microsoft.</value>
```

Comment:
typically these seem to say "anonymous usage data" -- assuming that's true. (there will be a right answer, maybe whaty ou have is the right answer)